lib/static/components/header/summary/dbBtn.jsx

```diff
@@ -11,16 +10,14 @@ const ForwardedDbBtn = React.forwardRef(function DbBtn({fetchDbDetails}, ref) {
     const content = `Databases loaded: ${value}`;
 
     return (
-        <Ref innerRef={ref}>
-            <Button
-                view={isFailed ? 'flat-danger' : 'flat'}
-            >
-                <div className='db-info'>
-                    <ChevronDown/>
-                    {content}
-                </div>
-            </Button>
-        </Ref>
+        <Button
```

Comment:
`<Ref>` is deprecated.

Reply:
i recall it was usefull when it was wrapping semantic-ui-react component, as they can't pass `ref` the usual way. Useless with gravity-ui, though

Reply:
Semantic UI supports ref forwarding too, as of v3